handwrite/default_multiple.json

```diff
@@ -0,0 +1,348 @@
+{
+  "props": {
+    "ascent": 800,
+    "descent": 200,
+    "em": 1000,
+    "encoding": "UnicodeFull",
+    "lang": "English (US)",
+    "filename": ["FirstFont", "SecondFont"],
+    "style": ["Regular", "Bold"]
+  },
+  "sfnt_names": [
+    {
+      "Copyright": [
+        "Copyright (c) 2021 by Nobody",
+        "Copyright (c) 2021 by Somebody"
+      ]
+    },
+    { "Family": ["FirstFont", "SecondFont"] },
+    { "SubFamily": ["Regular", "Bold"] },
+    { "UniqueID": ["FirstFont 2021-02-04", "FirstFont 2021-02-09"] },
+    { "Fullname": ["FirstFont Regular", "SecondFont Bold"] },
+    { "Version": ["Version 001.000", "Version 001.000"] },
+    { "PostScriptName": ["FirstFont-Regular", "SecondFont-Bold"] }
+  ],
```

Comment:
Since this is default config, you can't assume 
1. There are multiple sheets
2. The second sheet to be bold. Let's keep it MyFont2

We might need to change Metadata to something related to filename if config is not passed explicitly.

Reply:
K